ui/src/tool-panel/location-track/location-track-infobox.tsx

```diff
@@ -157,7 +157,7 @@ const LocationTrackInfobox: React.FC<LocationTrackInfoboxProps> = ({
                 visibilityChange={visibilityChange}
                 setConfirmingDraftDelete={setConfirmingDraftDelete}
             />
-            {layoutContext.branch == 'MAIN' && (
+            {layoutContext.branch === 'MAIN' && layoutContext.publicationState === 'OFFICIAL' && (
```

Comment:
GVT-2767: Piilotetaan Vie Ratkoon -nappi kun ollaan luonnostilassa. Mietin pitkään kuuluisiko nappi piilottaa vai vaan disabloida, mutta a) kyseessä on jo lähtökohtaisesti aika advanced ja vähän niinkuin piilotoiminto ja b) suunnitelmatiloissa nappia ei haluta näyttää ollenkaan, niin pistin sen nyt näin.